bitrise.yml

```diff
@@ -2,27 +2,31 @@ format_version: '8'
 default_step_lib_source: 'https://github.com/bitrise-io/bitrise-steplib.git'
 project_type: ios
 pipelines:
-  pipeline_multiple_shards:
+  pipeline_build_and_test:
     stages:
-    - stage_1: {}
-    - stage_2: {}
+    - build_application: {}
+    - run_tests: {}
 stages:
-  stage_1:
+  build_application:
     workflows:
-    - configure_build_fennec: {}
-  stage_2:
+    - configure_build_fennec:
+        run_if: '{{getenv "BUILD_FIREFOX_IOS" | eq "true"}}'
```

Comment:
This is the correct place to run a workflow conditionally: https://devcenter.bitrise.io/en/builds/build-pipelines/configuring-a-bitrise-pipeline.html#setting-up-run_if-conditions-for-optional-workflows-using-pipelines